src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableIssue17036.java

```diff
@@ -0,0 +1,31 @@
+/*
+UnusedLocalVariable
+allowUnnamedVariables = false
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.unusedlocalvariable;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.nio.file.Files;
+
+public class InputUnusedLocalVariableIssue17036 {
+    static {
+        try (OutputStream os = Files.newOutputStream(null)) {
+            System.out.println(os);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        try {
+            OutputStream os = Files.newOutputStream(null); // violation, unused variable 'os'
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        try (OutputStream os = Files.newOutputStream(null)) { // violation, unused variable 'os'
+            System.out.println("os");
```

Comment:
```
[ERROR] Failures: 
[ERROR]   UnusedLocalVariableCheckTest.testIssue17036:589->AbstractModuleTestSupport.verifyWithInlineConfigParser:260->AbstractModuleTestSupport.verifyViolations:614 Violation lines for /home/vsts/work/1/s/src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unusedlocalvariable/InputUnusedLocalVariableIssue17036.java differ.
missing (1): 25
---
expected   : [21, 25]
but was    : [21]
```

do you mean that we missing to report violation on line 25 ? sounds like valid defect in Check , false-negative.

Reply:
yes have to check what you mean with CLI. the message output message is not important right? 
This here is easy starter to overtake and fix.

If i cant provide maybe someone else can supplement.